site/source/tutorials/TypeScript_and_Dojo_2.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: false
+title: TypeScript and Dojo 2
+overview: The TypeScript language provides a strong foundation that Dojo 2 has built upon to meet its goals. This article highlights Dojo 2's objectives and describes how TypeScript has helped those objectives become a reality.
+---
+# TypeScript and Dojo 2
+
+## What is TypeScript?
+
+According to [TypeScript's website](http://www.typescriptlang.org/index.html), three of TypeScript's major strengths are:
+
+- It starts and ends with JavaScript - TypeScript has not tried to re-implement the JavaScript language. Rather, it has been been built on top of existing standards so that all valid JavaScript is also valid TypeScript. Additionally, when a TypeScript project is compiled, it is converted into clean, simple JavaScript that will run in almost any JavaScript environment.
+- It has strong tools for large apps - TypeScript, as the name implies, adds an optional type-system on top of the JavaScript language. Types allow tools to be created that provide many advantages that other strongly typed language enjoy such as refactoring support, autocompletion functionality, and static code analysis.
```

Comment:
It feels like before we say that Types allow tools, we should say something about it making it easier to reason about code, but I understand this comes straight from TS?